lib/rules/forbid-prop-types.js

```diff
@@ -181,6 +181,14 @@ module.exports = {
       },
 
       CallExpression(node) {
+        if (
+          node.callee.object
+          && !propsUtil.isPropTypesDeclaration(node.callee)
+          && node.callee.object.name !== 'PropTypes'
```

Comment:
can we ensure this is just `React.PropTypes`, or that `PropTypes` is imported from `prop-types` or `React`? otherwise if someone names anything `PropTypes` it'd bypass this rule.

Reply:
Let's add two test cases - one that imports react as something that's not React, and one that imports prop-types as something that's not PropTypes - and that might allow us to remove this line?

Reply:
The first one has `import { any } from "prop-types"; `?

Reply:
ohh hm, right. in that case, i think we should be setting the propTypesPackageName to the empty string maybe? or, maybe we need to track a list of all the named imports from `prop-types` as well?

Reply:
i'm not really sure.

Basically, the common pattern is `import PropTypes from 'prop-types'; … PropTypes.something`, or `import React from 'react'; … React.PropTypes.something`. However, if someone does `import { something } from 'prop-types'`, then `something` should still be able to be forbidden.